requirments.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
+numpy
+datetime
+python-amazon-product-api
```

Comment:
Yeah I was wondering the same as I couldn't get the code to work on my machine. Thanks for that fix